cliquet/tests/test_cache.py

```diff
@@ -156,12 +159,86 @@ def test_ping_returns_false_if_unavailable(self):
     def test_ping_logs_error_if_unavailable(self):
         pass
 
+    def test_cache_prefix_is_set(self):
+        settings_prefix = self.settings.copy()
+        settings_prefix['cache_prefix'] = 'prefix_'
+        config_prefix = self._get_config(settings=settings_prefix)
+
+        # initiating cache backend with prefix:
+        backend_prefix = self.backend.load_from_config(config_prefix)
+
+        # set a value with a cache that has no prefix
+        backend_prefix.set('key', 'foo')
+
+        # obtain the value of cache that has prefix
+        obtained = backend_prefix.get('key')
+
+        self.assertEqual(obtained, 'foo')
```

Comment:
This idea is really good, but doesn't work with the MemoryBackend because they are not sharing the same store.